LED_BUTTONS/app.c

```diff
@@ -0,0 +1,31 @@
+#include"Macros.h"
+#include"TypeDef.h"
+#include"DIO_retval.h"
+#include"DIO.h"
+#include"Button_retval.h"
+#include"Button.h"
+#include"LED.h"
+#include<util/delay.h>
+
+int main() {
+	LED_Init(PORTA, PIN7);
+
+	uint8_t state = 0;
+	uint8_t PushedButton = 1;
+	state = Button_Init(PORTA, PIN4);
+
+	while (1) {
+		PushedButton=Button_getState(PORTA, PIN4);
+		_delay_ms(20);
+		if (PushedButton == 0) {
+			PushedButton=Button_getState(PORTA, PIN4);
+			if (PushedButton == 1) {
+				LED_toggle(PORTA, PIN7);
+			}
+		}
+
+
+	}
+
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of c files

[Sprints_CodeGuideLine.pdf](https://github.com/KhalidMosad/Sprints-Tasks/files/6901176/Sprints_CodeGuideLine.pdf)

![4](https://user-images.githubusercontent.com/44152298/127508098-dd2316ae-1851-429c-80c2-ae418e29dd28.PNG)